### PR TITLE
ME-62 - Center DarkNavigationButton content

### PR DESCRIPTION
**Pull request by kierangillen**
- I'd really love to change the name of this component down the road too if we are keeping it.
- https://artsyproduct.atlassian.net/browse/ME-62

## Before

Screen Shot 2019-10-17 at 12 25 50 PM

## After

Screen Shot 2019-10-17 at 1 05 04 PM

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -70,5 +70,5 @@
     "editor.formatOnSave": false
   },
   "javascript.validate.enable": true,
-  "debug.node.autoAttach": "on"
+  "debug.node.autoAttach": "off"
```
Comment (ashfurrow) [inline .vscode/settings.json]: Was the intentionally changed?

File: src/lib/Components/Buttons/DarkNavigationButton.tsx [hunk 2]
```diff
@@ -13,37 +10,20 @@ interface Props extends React.Props<DarkNavigationButton> {
...
-const Row = styled.View`
-  flex-direction: row;
-  justify-content: space-between;
-`
-
 export default class DarkNavigationButton extends React.Component<Props, any> {
   render() {
     const showNavArrow = this.props.href || this.props.onPress
     return (
-      <BackgroundView style={this.props.style}>
```
Comment (ashfurrow) [inline src/lib/Components/Buttons/DarkNavigationButton.tsx]: Seems like we're not using the `style` prop anymore. Will that affect anywhere else that's using this component? Should we remove it from the `Props` definition above?

**Comment by kierangillen** [comment]
@ashfurrow Ah good points, updating now, thanks.

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,6 +17,7 @@
 ### Master
 
 - Add graphql request duration reporting via volley - ds300
+- Centers DarkNavigationButton - kierangillen
```
Comment (ashfurrow) [inline CHANGELOG.md]: Ah! One last suggestion:

```suggestion
- [ME-62] Centers DarkNavigationButton - kierangillen
```

**Comment by artsyit** [comment]
:rocket: PR was released in v1.18.1 :rocket:

**Pull request merged**